src/grpc.ts

```diff
@@ -25,13 +25,13 @@ export interface GoogleAdsNodeOptions {
 }
 
 export default class GrpcClient {
-    private client: GoogleAdsClient
+    private readonly client: GoogleAdsClient
 
     constructor(
         developer_token: string,
         client_id: string,
         client_secret: string,
-        refresh_token: string,
+        private readonly refresh_token: string,
```

Comment:
I don't think `private` is required here?

Reply:
heh, yep you right, it's not required :) thank you for catching it